scripts/framework_env.py

```diff
@@ -25,7 +25,8 @@ def __init__(self, logger, ndict=None, verbose=0, clean=False,
                  host_files=None, scheme_files=None, suites=None,
                  preproc_directives=[], generate_docfiles=False, host_name='',
                  kind_types=[], use_error_obj=False, force_overwrite=False,
-                 output_root=os.getcwd(), ccpp_datafile="datatable.xml"):
+                 output_root=os.getcwd(), ccpp_datafile="datatable.xml",
+                 debug = False):
```

Comment:
Might be pedantic but for consistency:
```suggestion
                 debug=False):
```